helm/aws-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,38 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-operator.name" -}}
+{{- default .Chart.Name .Values.project.name | trunc 63 | trimSuffix "-" -}}
```

Comment:
What does this default do?

Reply:
It takes `.Values.project.name` if defined, otherwise just `.Chart.Name`, truncates it to 63 characters to comply with k8s resource name limits and then also strips a trailing `-` in case the truncation happened to result in it.

This is taken from: https://github.com/helm/helm/blame/ec1d1a3d3eb672232f896f9d3b3d0797e4f519e3/pkg/chartutil/create.go#L343

Reply:
When we use this helper maybe we can delete `Values.project.name` altogether?

Reply:
Do you mean replace it with `.Values.nameOverride` as in the Helm helper? That works for me.

Reply:
We discussed that. We'll drop `.Values.project` altogether. There is no need for this.

Reply:
Done